packaging/flatpak/generate-flatpak-script.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+DIR=`dirname "$(readlink -f "$0")"`
+
+cd $DIR
+
+python3 -m venv flatpak-env
+export PATH=$DIR/flatpak-env/bin:$PATH
+
+cat requirements.flatpak.txt ../../requirements.txt > requirements.txt
+
+flatpak-env/bin/pip install -Ur requirements.txt
+
+# freeze requirements and ignore blacklisted packages
+flatpak-env/bin/pip freeze | grep -v PyYAML | grep -v six= | grep -v matrix-nio > flatpak-requirements.txt
+
+# generate flatpak requirements
+flatpak-env/bin/python flatpak-pip-generator --output flatpak-pip --requirements-file=flatpak-requirements.txt
```

Comment:
Lines should stay under 80 columns if possible, unless there's something like a long URL string that can't be cut easily